Trigger/src/ReadTriggerInfo_module.cc

```diff
@@ -1161,7 +1161,7 @@ namespace mu2e {
 	  if ( (moduleLabel.find("caloMVACEFilter")!= std::string::npos) || 
 	       (moduleLabel.find("TSFilter")       != std::string::npos) ||
 	       isCosmicHelix ){ 
-	    findTrigIndex(_trigFinal, moduleLabel, index);
+	    //	    findTrigIndex(_trigFinal, moduleLabel, index);
```

Comment:
Why are these lines commented?  Are you intending to restore them in some future commit?  If not, please just delete them.